### PR TITLE
Answer Modal & Add a Question

### PR DESCRIPTION
**Pull request by afong888**
Combined both tickets in to one PR
Similar changes to both question and answer modals

https://trello.com/c/vWSpivkt
https://trello.com/c/W8Epitvy

### CODE REVIEW

**Comment by r-jeng** [comment]
Picking up ticket.

File: client/src/components/Q&A/AnswerModal.jsx [hunk 1]
```diff
@@ -24,28 +54,68 @@ const AnswerModal = (props) => {
...
+            &#34;
+            {productName}
+            :&#160;
+            {question}
+            &#34;
+          </span>
         </div>
+        <form>
+          <div className="modal-body">
+            <span>Your Answer:</span>
```
Comment (r-jeng) [inline client/src/components/Q&A/AnswerModal.jsx]: Awesome job on the Answer Modal! There seems to be a part in the business requirements that would like an asterisk next to the title where there should be a fill out.

"...Each should be labelled as titled below. Those indicated as mandatory should have an asterisk next to the title." 
1.3.6.1. Your Answer (mandatory)
1.3.6.2. What is your nickname (mandatory)
1.3.6.3. Your email (mandatory)

File: client/src/components/Q&A/QuestionModal.jsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+          <span>
+            &#34;
+            About the&#160;
+            {productName}
+            &#34;
+          </span>
+        </div>
+        <form>
+          <div className="modal-body">
+            <span>Your Question:</span>
```
Comment (r-jeng) [inline client/src/components/Q&A/QuestionModal.jsx]: Also great job on the Question Modal! There also seems to be a part in the business requirements that would like an asterisk next to the title if a fill out is mandatory.

"...Each should be labelled as titled below. Those indicated as mandatory should have an asterisk next to the title."
1.3.5.1. Your Question (mandatory)
1.3.5.2. What is your nickname (mandatory)
1.3.5.3. Your email (mandatory)

File: client/src/components/Q&A/questions.css [hunk 3]
```diff
@@ -15,14 +15,14 @@
 }
 
 .search-question {
-  background-color: black;
+  background-color: grey;
```
Comment (r-jeng) [inline client/src/components/Q&A/questions.css]: It's a little difficult to read the search question placeholder text here. The text seems to blend into the grey background.

**Review by r-jeng — CHANGES_REQUESTED** [review]
Nice work on the Answer & Question Modals! Functionality for both modals are working great, I just added a few comments. Could you help make these minor change requests? Let me know if you have any questions!

File: client/src/components/Q&A/AnswerModal.jsx (hunk 1 above)
Comment (afong888) [reply client/src/components/Q&A/AnswerModal.jsx]: edited

File: client/src/components/Q&A/QuestionModal.jsx (hunk 2 above)
Comment (afong888) [reply client/src/components/Q&A/QuestionModal.jsx]: edited

File: client/src/components/Q&A/questions.css (hunk 3 above)
Comment (afong888) [reply client/src/components/Q&A/questions.css]: removed search questions bar and css related to it

**Comment by r-jeng** [comment]
Thanks for making the changes. Merged to main!

**Pull request merged**